arbnode/dataposter/data_poster.go

```diff
@@ -112,13 +111,13 @@ func NewDataPoster[Meta any](db ethdb.Database, headerReader *headerreader.Heade
 	}, nil
 }
 
-func (p *DataPoster[Meta]) Sender() common.Address {
+func (p *DataPoster) Sender() common.Address {
 	return p.sender
 }
 
-func (p *DataPoster[Meta]) GetNextNonceAndMeta(ctx context.Context) (uint64, Meta, error) {
+func (p *DataPoster) GetNextNonceAndMeta(ctx context.Context) (uint64, []byte, error) {
 	config := p.config()
-	var emptyMeta Meta
+	var emptyMeta []byte
```

Comment:
emptyMeta was used here because we couldn't return `nil` for Meta. We should replace emptyMeta with nil in the return statements

Reply:
Done. (Bulk replaced Meta with []byte when I was refactoring). Thanks for spotting.